### ISSUE TITLE
Environment variables like LDAP_BINDDN not copying proprtly yo config.inc.php

### ISSUE THREAD

**Issue by JustoStyle** [tiredofit/docker-self-service-password#23]
Hi,

It seems with environment variables like LDAP_BINDDN. That even when escaped the value gets cut off at the first space. e.g

LDAP_BINDDN=CN=Service User,OW=Service,DC,Domain etc
will end up in config like 
$ldap_binddn = "CN=Service"

**Comment by tiredofit** [comment]
OK makes sense, and if you edit the config.php manually with your entry does it work?

**Comment by JustoStyle** [comment]
Yeah it does. It work in previous versions too. I only updated 4 days ago.

**Comment by tiredofit** [comment]
OK. Related to recent changes. Will add a fix early tomorrow. Sorry about this.

**Comment by tiredofit** [comment]
New release `tiredofit/self-service-password:5.0.2` or `latest` that should allow you to have spaces in LDAP_BINDDN, LDAP_BASE_SEARCH and LDAP_FILTER environment variables.

**Comment by JustoStyle** [comment]
Tested all looks to be working.

Thank you.
